pkg/ragengine/manifests/manifests.go

```diff
@@ -17,7 +17,7 @@ import (
 
 var controller = true
```

Comment:
Can we also remove this controller variable?

Reply:
fixed, thanks